gui/tracker.py

```diff
@@ -1659,6 +1669,25 @@ def on_click_inventory_button(self, item: Item, item_image: str):
         if self.last_checked_location is not None and item is not None:
             self.last_checked_location.tracked_item = item
             self.last_checked_location.tracked_item_image = item_image
+            # If this is a crystal, ask the user if it was 1 or 5
+            if item.name == GRATITUDE_CRYSTAL:
+                single_crystal_question = FiQuestionDialog(self.main)
+                single_crystal_question.show_dialog(
+                    "Gratitude Crystal Amount?",
+                    "Did you find 1 crystal or 5 crystals?",
+                    "1 crystal",
+                    "5 crystals",
+                )
+                if single_crystal_question != QMessageBox.StandardButton.Yes:
+                    item = self.world.get_item(GRATITUDE_CRYSTAL_PACK)
+                    crystals_button = self.gratitude_crystals_button
+                    for _ in range(4):
+                        crystals_button.state += 1
+                        if crystals_button.state >= len(crystals_button.items):
+                            crystals_button.state = 0
```

Comment:
We need to make sure the extra crystals are added to the inventory too.
```suggestion
                    for _ in range(4):
                        crystals_button.state += 1
                        crystals_button.add_current_item()
                        if crystals_button.state >= len(crystals_button.items):
                            crystals_button.state = 0
                            crystals_button.remove_all_items()
```